docs/en/framework/ui/mvc-razor-pages/navigation-menu.md

```diff
@@ -169,9 +169,9 @@ context.Menu.GetAdministration().AddItem(...)
 
 ### Manipulating the Existing Menu Items
 
-ABP Framework executes the menu contributors by the [module dependency order](../../Module-Development-Basics.md). So, you can manipulate the menu items that your application or module (directly or indirectly) depends on.
+ABP Framework executes the menu contributors by the [module dependency order](../../architecture/modularity/basics.md). So, you can manipulate the menu items that your application or module (directly or indirectly) depends on.
 
-**Example: Set an icon for the `Users` menu item added by the [Identity Module](../../Modules/Identity.md)**
+**Example: Set an icon for the `Users` menu item added by the [Identity Module](.././../../../../../modules/identity.md)**
```

Comment:
Should be `../../../modules/identity.md`

Reply:
**Example: Set an icon for the `Users` menu item added by the [Identity Module](.././../../../../../modules/identity.md)**